### ISSUE TITLE
Blue Artifact Appears Inside Globe When Rotating with .showAtmosphere Enabled

### ISSUE THREAD

**Issue by shreejalmaharjan-27** [vasturiano/globe.gl#164]
**Describe the bug**
I have encountered an unusual issue while interacting with the globe. When I rotate it, a strange blue artifact appears inside the globe. This problem was not present in previous versions of the software. After some investigation, I have identified that the culprit is the ".showAtmosphere" feature. When this feature is disabled, no artifacts are visible.

**To Reproduce**
1. Open the globe application. https://stackblitz.com/edit/nuxt-starter-mpegr1?file=components%2FGlobe.vue
2. Turn the globe, a weird blue artifact appears inside the globe
3. add .showAtmosphere(false) to the world object, the artifact no longer appears

**Expected behavior**
When rotating the globe with the ".showAtmosphere" feature enabled, there should be no artifacts or anomalies observed.

**Screenshots**
![image](https://github.com/vasturiano/globe.gl/assets/58314282/6cb1c0cf-dd13-4100-9099-b5f23ae16467)

Upon rotating the globe with the ".showAtmosphere" feature enabled, a noticeable blue artifact appears inside the globe, which is not the expected behavior.

**Desktop (please complete the following information):**
 - OS: Windows/Mac
 - Browser: Chrome, Firefox
 - Version: Latest

**Smartphone (please complete the following information):**
 - Device: not tested
 - OS:
 - Browser: 
 - Version:

**Comment by vasturiano** [comment]
@shreejalmaharjan-27 thanks for reaching out.

I'm unable to run your repro link. But I'm going to guess that your globe's surface is transparent (at least partially where the water should be). Is that right? If so, this outcome makes sense even though it may not be obvious why. The atmosphere visual effect is essentially an additional sphere inside the globe covered in a diffuse material, to create the illusion of radiation just above the surface of the globe. Normally you don't see the part inside the globe because it's being hidden by the globe's surface, but if you add a transparent surface it will become visible. This is probably what you're seeing.

The TLDR is that atmosphere on a hollow globe doesn't really go together well visually.

**Comment by shreejalmaharjan-27** [comment]
Indeed, my globe is transparent. Previously using the same configuration I had achieved a translucent effect where the water should be, and it looked cool. I was wondering if it's possible to disable the water effect.  Also, the stackblitz preview is only visible when you open it in a new tab.
